front/src/applications/editor/tools/pointEdition/components.tsx

```diff
@@ -328,7 +308,20 @@ export const PointEditionLeftPanel: FC<{ type: EditoastType }> = <Entity extends
             additionalUpdate.geometry = point.geometry;
           }
 
-          setState({ ...state, entity: { ...(entity as Entity), ...additionalUpdate } });
+          if (entity.objType === 'Signal' && entity.properties.logical_signals) {
+            additionalPropertiesUpdate.logical_signals = formatSignalingSystems(
+              entity as unknown as SignalEntity
+            );
+          }
+
```

Comment:
entity cannot be typed as EditorEntity ?

Reply:
No because we know we have a signal, and we need to give a SignalEntity to the function `formatSignalingSystems`